src/components/input/Select.tsx

```diff
@@ -250,6 +254,11 @@ SelectOption.displayName = 'Select.Option';
 
 /** Small space to apply between the toggle button and the listbox */
 const LISTBOX_TOGGLE_GAP = '.25rem';
+/**
+ * Space in pixels to apply between the listbox and the viewport sides when the
+ * listbox grows to the very edges
+ */
+const LISTBOX_VIEWPORT_GAP = 8;
```

Comment:
Is this the horizontal gap, vertical gap, or both?

Reply:
horizontal only. The listbox have a max vertical height after which it has it's own scroll.

In the rare occasions that doesn't fit the viewport height, the main scroll would allow you to see all items, which is not perfect, but has worked well so far.

Reply:
I have renamed this to `LISTBOX_VIEWPORT_HORIZONTAL_GAP`.